module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,13 @@
+#include "sort.hpp"
+
+std::deque<std::string> sort(std::forward_list<std::string> sortMe) {
+    sortMe.sort([](const std::string& lhs, const std::string& rhs) {
+        if (lhs.length() == rhs.length()) {
+            return lhs <= rhs;
+        }
+
+        return lhs.length() <= rhs.length();
```

Comment:
it would be better to use `<` instead of `<=`, because if elements are equal, sort algorithm does not have to move them.